python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -411,6 +411,48 @@ def test_subquery_alias(self) -> None:
         )
         self.assertTrue("special_alias" in plan_text)
 
+    def test_sort(self):
```

Comment:
these tests fails before this PR, we should add more e2e tests.

Reply:
big +1